testing/collect_ap_hwpl.py

```diff
@@ -19,6 +19,7 @@ def command_parser():
     parser.add_argument("-test_cases_path", default=default_test_cases_path, type=str, help="path of to intermediate files of the testcases")
     default_output_file=os.path.join(os.getcwd(), "hpwls.csv")
     parser.add_argument("-output_file", default=default_output_file, type=str, help="path of to output file")
+    parser.add_argument("-run_simulated_annealing", action="store_true", help="test simulated annealing")
```

Comment:
I would add a bit more to explain what this is doing. It can be a bit confusing since its not actually running Simulated Annealing. It may seem obvious now, but months down the line this can be confusing.